main.go

```diff
@@ -233,8 +236,9 @@ func mainRun(cmd *cobra.Command, args []string) {
 		DNSmasqTemplate:          globalFlags.dnsmasqTemplate,
 		TFTPRoot:                 globalFlags.tFTPRoot,
 		NoTLS:                    globalFlags.noTLS,
-		HTTPPort:                 globalFlags.httpPort,
-		HTTPBindAddress:          globalFlags.httpBindAddress,
+		APIPort:                  globalFlags.apiPort,
+		PXEPort:                  globalFlags.pxePort,
+		BindAddress:              globalFlags.bindAddress,
```

Comment:
Bind address is the ip address which will be used for listening socket (ie `127.0.0.1` or `10.0.4.200`), port is the TCP port used for listening  (ie 4080)

if you put them together you get `10.0.4.200:4080`